incubator/etcd/values.yaml

```diff
@@ -7,9 +7,13 @@ Name: etcd
 PeerPort: 2380
 ClientPort: 2379
 Component: "etcd"
+PeerTLS: true
+# Replicas scales the cluster with the "helm upgrade" command
+# The recommended etcd cluster size is 3, 5 or 7, which is decided by the fault tolerance requirement.
+# A 7-member cluster can provide enough fault tolerance in most cases.
+# While larger cluster provides better fault tolerance the write performance reduces since data needs to
+# be replicated to more machines.
 Replicas: 3
-Image: "gcr.io/google_containers/etcd-amd64"
-ImageTag: "2.2.5"
```

Comment:
again what is the reason for this? what if I wanted to use another image such as my own?

Reply:
See my answer above.